app/views/updateCase/UpdateAdditionalInformationView.scala.html

```diff
@@ -39,7 +39,8 @@
                 true,
                 true,
                 Some(messages("updateAdditionalInformation.hint")),
-                classes = Some("govuk-fieldset__legend")
+                classes = Some("govuk-fieldset__legend"),
+                maxCharacters = Some(1400)
```

Comment:
To maintain consistency, should we retrieve this value from UpdateAdditionalInformationFormProvider.maxLength?